### ISSUE TITLE
{% endihighlight %} fails to close the </code>

### ISSUE THREAD

**Issue by EmilStenstrom** [bdesham/inline_highlight#7]
I'm using this code inside a markdown file:

```
{% ihighlight html %}h1{% endihighlight %} after
```

And the output i get form inline_highlight is this HTML:
```html
<code class="highlight language-html" data-lang="html">h1{% endihighlight %} after</code>
```

... which is not correct as the tag should close before the endihighlight tag.

```bash
$ jekyll --version
jekyll 3.6.2
```

**Comment by bdesham** [comment]
Hmm, I’m unable to reproduce this. I created a minimal example site with _config.yml:

```
plugins: [jekyll_inline_highlight]
```

and test.md:

```
---
---

{% ihighlight html %}h1{% endihighlight %} after
```

but the resulting test.html looks right:

```
<p><code class="highlight language-html" data-lang="html">h1</code> after</p>
```

This is also on Jekyll 3.6.2. Do you have anything else in your configuration that might be causing this? A nonstandard Markdown parser, maybe?

**Comment by EmilStenstrom** [comment]
Thanks for testing this with a minimal example. You are right, and I am wrong. I'm in the process of migrating an old wordpress blog, and there I've had `<code>` and `</code>` blocks to show what should be highlighted. Now I've tried replacing that code with `{% ihighlight %}` and quoted blocks to get something to work. Problem is, I've managed to replace `</code>` with a quote **inside of the _plugins directory too**. So when I wanted to make sure I hade the exact same version of jekyll_inline_highlight.rb as you, I found that my version missed the </code> block that closes highlight blocks. Makes sense, but definitely not a problem with the plugin.

Sorry for taking your time!
